### DISCUSSION TITLE
Searchable dropdpown in inspector

### DISCUSSION THREAD

**Discussion by kvinita** [clientIO/joint#2586] [Q&A]
### Introduction

I have a dropdown in inspector that has 100+ options. Is there a way to create a searchable dropdown? 
Something like this:
![image](https://github.com/clientIO/joint/assets/124018900/c618a998-86a8-4f4b-91fe-b8576e45aee8)

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
Please see our example on custom *Inspector* fields:
https://resources.jointjs.com/docs/rappid/v4.0/ui.Inspector.html#ui.Inspector.custom-fields

<img width="256" alt="image" src="https://github.com/clientIO/joint/assets/3967880/ec5c72c2-c142-467c-92dd-9fe6f4d4cc0d">
